src/app/model-download/components/model-download-component/model-download-component.component.html

```diff
@@ -0,0 +1,231 @@
+                                                stroke-width="2" d="M5 13l4 4L19 7" />
+                                        </svg>
+                                    </div>
+                                </div>
+                                <div *ngIf="model.status === 'failed'" data-tip="Embedding creation ran into errors." class="tooltip">
+                                    <div class="btn cursor-default btn-circle btn-xs no-animation btn-error mx-2">
+                                        <svg xmlns="http://www.w3.org/2000/svg"
+                                            class="inline-block w-4 h-4 stroke-current" fill="none"
+                                            viewBox="0 0 24 24" stroke="currentColor">
+                                            <path stroke-linecap="round" stroke-linejoin="round"
+                                                stroke-width="2"
+                                                d="M12 9v2m0 4h.01m-6.938 4h13.856c1.54 0 2.502-1.667 1.732-3L13.732 4c-.77-1.333-2.694-1.333-3.464 0L3.34 16c-.77 1.333.192 3 1.732 3z" />
+                                        </svg>
+                                    </div>
+                                </div>
+                                <div *ngIf="model.status === 'downloading'" data-tip="Model is downloading" class="tooltip">
+                                    <kern-loading color="yellow" size="btn-xs"></kern-loading>
+                                </div>
+                                <div *ngIf="model.status === 'initializing'" data-tip="Model is initializing" class="tooltip">
+                                    <svg xmlns="http://www.w3.org/2000/svg" class="icon icon-tabler icon-tabler-loader" width="24" height="24" viewBox="0 0 24 24" stroke-width="2" stroke="currentColor" fill="none" stroke-linecap="round" stroke-linejoin="round">
+                                        <path stroke="none" d="M0 0h24v24H0z" fill="none"></path>
+                                        <line x1="12" y1="6" x2="12" y2="3"></line>
+                                        <line x1="16.25" y1="7.75" x2="18.4" y2="5.6"></line>
+                                        <line x1="18" y1="12" x2="21" y2="12"></line>
+                                        <line x1="16.25" y1="16.25" x2="18.4" y2="18.4"></line>
+                                        <line x1="12" y1="18" x2="12" y2="21"></line>
+                                        <line x1="7.75" y1="16.25" x2="5.6" y2="18.4"></line>
+                                        <line x1="6" y1="12" x2="3" y2="12"></line>
+                                        <line x1="7.75" y1="7.75" x2="5.6" y2="5.6"></line>
+                                    </svg>
+                                </div>
+                            </td>
+                            <td class="whitespace-nowrap text-center px-3 py-2 text-sm text-gray-500">
```

Comment:
creating a delete modal for every row works but stresses the dom. For now this is fine, in the future we should try to use one modal at the end of the component & a variable reference for better readability and less dom clutter (see project settings #labelEditCheckbox and the related modal for reference)

Reply:
I added it at the end of the component and set the value on click of the delete icon